did-configuration/demo/scripts/create_did_configuration.js

```diff
@@ -0,0 +1,54 @@
+const fs = require("fs");
+const path = require("path");
+const ES256K = require("@transmute/es256k-jws-ts");
+
+// feel free to impersonate me :/
```

Comment:
https://github.com/WebOfTrustInfo/btcr-hackathon-2019/issues/6

I didn't have to link a leaked key, but I did.

this key is used all over the place in my ES256K tests, I will need to revoke this key at some point, and generate a clean set of claims for the demo.